packages/curve-ui-kit/src/shared/ui/InvertOnHover.tsx

```diff
@@ -37,36 +32,8 @@ type InvertOnHoverProps = {
   disabled?: boolean
 }
 
-const defaultHoverColor = (t: Theme) => t.design.Layer.TypeAction.Hover
-
-export const InvertOnHover = ({
```

Comment:
RIP 💀 

I'm not sure if we really want to remove it like that, given we also use it for the token selector. Maybe consider adding a stupid boolean instead?

![image](https://github.com/user-attachments/assets/1e29791b-ad75-48e6-b45f-28dcea7935f6)

Reply:
Tutti asked me to remove there too for consistency